### PR TITLE
[TR] PIM-4528: Fix attribute field display on error during mass edit

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | ?
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | Y
| Fixed tickets        | PIM-4528
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig [hunk 1]
```diff
@@ -59,8 +59,8 @@
     {{ parent() }}
     <script type="text/javascript">
         require(
-            ['jquery', 'underscore', 'pim/scopable', 'pim/currencyfield', 'datepicker', 'oro/mediator', 'jquery.multiselect', 'jquery.multiselect.filter'],
-            function ($, _, Scopable, CurrencyField, datepicker, mediator) {
+            ['jquery', 'underscore', 'pim/scopable', 'pim/currencyfield', 'datepicker', 'oro/mediator', 'routing', 'jquery.multiselect', 'jquery.multiselect.filter'],
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig]: multiline array would be more readable

File: features/mass-action/edit_common_attributes.feature [hunk 2]
```diff
@@ -259,3 +259,21 @@ Feature: Edit common attributes of many products at once
...
+  Scenario: See previously selected fields on mass edit error
+    Given I mass-edit products boots and sandals
+    And I choose the "Edit common attributes" operation
+    And I display the Weight and Name attribute
+    And I change the "Weight" to "Edith"
+    And I move on to the next step
+    Then I should see "Product information"
+    And I should see "Weight"
+    And I should see "Name"
+    And I am on the attributes page
```
Comment (jmleroux) [inline features/mass-action/edit_common_attributes.feature]: 'When' or 'Given I am on the attributes page'

**Comment by jmleroux** [comment]
GTM after fixes

File: src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig [hunk 3]
```diff
@@ -116,6 +129,8 @@
                         var checked = _.map($el.multiselect('getChecked'), function(el) {
                             return +el.value;
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig]: I know it's not one of your modifications but, wtf is that ?

**Comment by phaseinducer** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig (hunk 3 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig]: No idea ! :/

**Pull request merged**